.github/workflows/build-test.yml

```diff
@@ -23,6 +23,10 @@ jobs:
         uses:                   actions/checkout@v2
         with:
           fetch-depth:          50
+      # https://github.com/actions/cache/issues/133
+      - name:                   Fixup the owner of ~/.cargo/
+        if:                     matrix.platform == 'ubuntu-16.04'
```

Comment:
You will probably also want to do this on macOS.

Reply:
It doesn't seem to be a problem for macOS.
Github uses a third-party service for macOS runners: https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#cloud-hosts-for-github-hosted-runners as opposed to Standard_DS2_v2 VMs on Asure for Linux and Windows.

Reply:
Example job on macOS: https://github.com/openethereum/openethereum/runs/506808782.